core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -1216,6 +1228,10 @@ private static String toSql(RelNode root, SqlDialect dialect,
         + "FROM \"foodmart\".\"product\"\n"
         + "GROUP BY ROLLUP(\"product_class_id\")\n"
         + "LIMIT 5";
+    final String expectedTrino = "SELECT \"product_class_id\", COUNT(*) AS \"C\"\n"
```

Comment:
Trino itself also supports limit, do we need to convert limit into fetch here, and keep same as presto?

Reply:
Other dialects would support fetch next syntax first if they supported, and the change here is different from presto, so I prefer keep it the same with other dialect.